ads/ix.md

```diff
@@ -16,22 +16,9 @@ limitations under the License.
 
 # Index Exchange
 
-Index supports both direct ad tags and Header Tag style bidding using Doubleclick as the ad server.
+Index supports Header Tag style bidding using Doubleclick as the ad server.
 
-## Examples
-
-### Ad tag ###
-
-```html
-<amp-ad width=300 height=250
```

Comment:
to make sure, the direct ad tags are not supported. If this is no longer a valid "ix" ad, please also remove it from the ads.amp.html examples.